contenttype.go

```diff
@@ -313,6 +314,40 @@ func (mediaType MediaType) MIME() string {
 	return stringBuilder.String()
 }
 
+// EqualsExactly checks whether the provided MIME media type matches this one
+// including all parameters
+func (mediaType MediaType) EqualsExactly(mt MediaType) bool {
```

Comment:
Maybe let's use just "Equals".

Reply:
I'll use Equal, which then causes it to also match the equity method lookup used by github.com/google/go-cmp.

And done.